bin/semver

```diff
@@ -56,6 +75,7 @@ function main () {
         break
       case "-h": case "--help": case "-?":
         return help()
+        break
```

Comment:
A break isn't strictly necessary here, since the `return` is just as effective at ending the block.  But if a linter or something is complaining about it, whatevdr.